EmbeddedPkg/EmbeddedPkg.dsc

```diff
@@ -218,7 +218,7 @@
   EmbeddedPkg/Library/CoherentDmaLib/CoherentDmaLib.inf
   EmbeddedPkg/Library/NonCoherentDmaLib/NonCoherentDmaLib.inf
   EmbeddedPkg/Library/DxeDtPlatformDtbLoaderLibDefault/DxeDtPlatformDtbLoaderLibDefault.inf
-!if $(TOOL_CHAIN_TAG) == GCC      # MU_CHANGE - Makes assumptions about GCC artifacts.
+!if $(TOOL_CHAIN_TAG) == GCC5     # MU_CHANGE - Makes assumptions about GCC artifacts.

```

Comment:
Were these intentionally changed back to GCC5?

Reply:
This is a little confusing because the commit message is:

"SecurityPkg/Tcg2Smm: Added communicate buffer check
Merged PR 5400
Added communicate buffer check for the Tcg2Smm driver in lieu of the legacy MmOutsideValid check."

But the commit is changing GCC tags in the DSC file.

Reply:
I must have done something wrong here.

I'll re create the cherry-picks

Reply:
Yes, it was an oversight. the changes associated were already included in #287 that enabled CI.